addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -832,6 +868,19 @@ public String toDPTid(Class<? extends Type> typeClass) {
         return defaultDptMap.get(typeClass);
     }
 
+    @Override
+    public Class<? extends GenericItem> toItemTypeClass(Class<? extends Type> typeClass) {
+        return defaulItemTypeMap.get(typeClass);
+    }
+
+    public Class<? extends GenericItem> toItemTypeClass(String dpt) {
+        return defaulItemTypeMap.get(toTypeClass(dpt));
+    }
+
+    public String toItemType(String dpt) {
```

Comment:
This is neither part of the public API (i.e. interface) nor is it used internally. Therefore this method is dead code and can be deleted.

Reply:
It is used in the advanced binding, and in that context its natural place is in the KNXCoreTypeMapper (it can be potentially be used by other knx binding derivaties), but if it is really an issue I will put it in an inheriting class.